salt/kubernetes-master/controller-manager.jinja

```diff
@@ -10,7 +10,8 @@ KUBE_CONTROLLER_MANAGER_ARGS="\
     --cluster-cidr={{ pillar['cluster_cidr'] }} \
     --service-account-private-key-file={{ pillar['ssl']['key_file'] }} \
     --root-ca-file={{ pillar['ssl']['ca_file'] }} \
-{% if salt['pillar.get']('infrastructure', 'libvirt') == 'aws' -%}
```

Comment:
AWS doesn't need a config file - it uses IAM directly, so the creds don't need to be in a file anywhere.